CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-2.6.0...master
 
+### Fixed
+- AuroraMultiReplicaConsistency#isConsistent always calls replica connection supplier
```

Comment:
I think it would be good to operate on consumer-level too., it's hard to understand what imply this.
Mentioned sth about not opening connections if not needed or
replica consuming too many connections

Reply:
I'm open to hearing proposals.